src/systems/config/labor_config.cairo

```diff
@@ -0,0 +1,65 @@
+#[system]
+mod CreateLaborCR {
+    use traits::Into;
+    use traits::TryInto;
+    use array::ArrayTrait;
+    use box::BoxTrait;
+    use eternum::utils::unpack::unpack_resource_ids;
+    use eternum::components::config::LaborCR;
+    use eternum::components::config::LaborCV;
```

Comment:
BTW we can now group imports, so `use eternum::components::config::{LaborCR, LaborCV}`.